packages/workflow/src/ai/ai.ts

```diff
@@ -108,12 +108,18 @@ ${before.text}
   }
 
   async execute() {
-    const openai = new OpenAI({
-      // biome-ignore lint/complexity/useLiteralKeys: biome bug
-      apiKey: process.env["OPENAI_API_KEY"],
-    });
+    const apiKey = process.argv
```

Comment:
lets use `yargs` for parsing process.argv (if there will be more then 1 arg, for now its fine to parse manually)